packages/headless/src/features/facets/range-facets/numeric-facet-set/numeric-facet-controller-actions.ts

```diff
@@ -28,11 +36,29 @@ export const executeToggleNumericFacetSelect = createAsyncThunk<
 >(
   executeToggleNumericFacetSelectType,
   (payload, {dispatch, extra: {validatePayload}}) => {
-    validatePayload(payload, {
-      facetId: facetIdDefinition,
-      selection: new RecordValue({values: numericFacetValueDefinition}),
-    });
+    validatePayload(payload, definition);
     dispatch(toggleSelectNumericFacetValue(payload));
     dispatch(executeToggleRangeFacetSelect(payload));
   }
 );
+
+/**
+ * Deselects the breadcrumb numeric facet value and then executes a search with the appropriate analytics tag.
+ * @param facetId (string) The unique identifier of the facet (e.g., `"1"`).
+ * @param selection (NumericFacetValue) The target numeric facet value.
+ */
+export const executeNumericFacetBreadcrumb = createAsyncThunk<
```

Comment:
Similar to above, I think `deselectNumericFacetBreadcrumb` is a clearer name. Alternatively, I do not know if these `controller-actions` files are useful anymore. We could place the dispatch sequence in the controller.